apps/addons/management/commands/approve_addons.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand, CommandError
+
+import amo
+from addons.models import Addon
+from editors.helpers import ReviewHelper
+
+
+class Command(BaseCommand):
+    args = '<addon_guid addon_guid ...>'
+    help = 'Approve a list of add-ons, given their GUIDs.'
+
+    def handle(self, *args, **options):
+        if len(args) == 0:  # No GUID provided?
+            raise CommandError(
+                'Please provide at least one add-on guid to approve.')
+
+        files = get_files(args)
+
+        if not files:
+            raise CommandError('No files found that need approval')
+
+        files_with_review_type = [
+            (file_, get_review_type(file_)) for file_ in files]
+
+        for file_, review_type in files_with_review_type:
+            print u'Addon {0}, version {1}, file {2}, review type {3}'.format(
```

Comment:
I could add a log, the aim for that was to show the changes to the person running the command, before the confirmation. Would you still like a log instead?